src/store/store.ts

```diff
@@ -0,0 +1,11 @@
+import { create } from 'zustand';
+import { UserStoreProps } from '@/shared/types/user/auth';
+
+ export const userStore = create<UserStoreProps>(() => ({
+  isAuthenticated: true,
+  accessToken: '',
+  username: '',
```

Comment:
@BardhMustafa would it be better to save a user object? something like user: {username: "admin", email: "admin@gmail.com"...}